README.md

```diff
@@ -1,6 +1,17 @@
 API Application for camptocamp.org v6
 =====================================
 
+Requirements
+------------
+
+ * Python 3.4
+ * PostgreSQL 9.4 / PostGIS 2.1
+ * virtualenv (1.7 or higher)
```

Comment:
I see that on the demo server ES 2.1.0 is still used. Should we upgrade to 2.3.2 on this server?